python/pylibcudf/pylibcudf/io/types.pyx

```diff
@@ -401,6 +401,14 @@ cdef class TableWithMetadata:
         """
         return self.metadata.per_file_user_data
 
+    @property
+    def num_rows_per_source(self):
```

Comment:
Could you add this new method to the typing stubs in `python/pylibcudf/pylibcudf/io/types.pyi`?

Reply:
Yup, done. Looking forward to type stub automation :)